testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java

```diff
@@ -163,8 +163,6 @@ private String getFakeClientUrl() {
         // we need to use some page which host exists – Firefox is throwing exceptions like crazy if we try to load
         // a page on a non-existing host, like e.g. http://non-existing-server/
         // also we need to do this here as getAuthServerRoot is not ready when firing this class' constructor
-        return getAuthServerRoot() + "auth/non-existing-page/?foo=bar";
-        // TODO replace ^^ with the following once KEYCLOAK-12173 and KEYCLOAK-12189 are resolved
-        // return getAuthServerRoot() + "auth/non-existing-page/?foo=bar&bar=foo#anchor";
+         return getAuthServerRoot() + "auth/non-existing-page/?foo=bar&bar=foo#anchor";
```

Comment:
Referring to this change ^^^

Reply:
when test passes doesn't that mean that this fix fixes both issues?

Reply:
We currently no way how to test [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) as i18n support was temporarily removed from the new Account Console and the test related to [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) was [ignored](https://github.com/keycloak/keycloak/blob/e731736d7e37d4e49279746d736c223c15da541c/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java#L130) for now. So even though we cannot confirm [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) was fixed by this PR, it's safe to do this change.